mintapi/api.py

```diff
@@ -514,32 +527,33 @@ def post(self, url, **kwargs):
         return self.driver.request('POST', url, **kwargs)
 
     def login_and_get_token(self, email, password, mfa_method=None,
-                            mfa_input_callback=None, headless=False,
-                            session_path=None, imap_account=None,
+                            mfa_input_callback=None,
+                            imap_account=None,
                             imap_password=None,
                             imap_server=None,
-                            imap_folder=None,
+                            imap_folder='INBOX',
```

Comment:
Thanks for this great work! Is this INBOX change intended to be a part of this PR and could you describe the change if so?

Reply:
This makes the change more consistent with the API it replaces. 'INBOX' was set as default before part of '__init__' function, but no longer, since __init__ is now only used to create selenium 'driver'.   See https://github.com/mrooney/mintapi/pull/220/files/c4bca605e950e21eaa58d84b734fa03c7afd9cea#diff-4117f091b0b8c6d18c2469a6f2452c55L433